ExampleModels/Arithmetic/src/arithmetic.jl

```diff
@@ -0,0 +1,107 @@
+module Arithmetic
+
+abstract type Expression end
+
+struct Constant <: Expression
+    value::Number
```

Comment:
Please use 2 spaces for indentation (this is just the preference that I like to use for my code bases, not a general rule)